### ISSUE TITLE
No audio while streaming a game

### ISSUE THREAD

**Issue by ligix** [ValveSoftware/steam-for-linux#11092]
#### Your system information

* Steam client version (build number or date): 1720650399
* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: [Yes/No] Yes (Steam Families Beta)
* Have you checked for system updates?: [Yes/No] Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/user-attachments/files/16227967/steam-logs.tar.gz)
* GPU: i7-1165G7 on client, amd rx 7600 on host 

#### Please describe your issue in as much detail as possible:
While streaming from a windows host to an arch linux client no audio can be heard, everything else works fine.

Audio is played on the correct speaker on the host (as it's the only one, I've unplugged everything else):
![windows_audio_mixer](https://github.com/user-attachments/assets/6a702970-284f-465f-ab70-3fa2f60386e8)

Nothing can be seen on pavucontrol:
![pavucontrol](https://github.com/user-attachments/assets/f9c600af-8bc1-4618-94b9-eed01d5fdafa)

So far I've tested with tf2, astroneer and just cause 3, all three do not work

#### Steps for reproducing this issue:

1. Launch steam on the host
2. Launch steam on the client
3. On the client navigate to the library, pick a game and click "stream"

**Comment by swimmadude66** [comment]
I am getting the exact same bug. Local play works fine, but remote play doesn't even create an audio stream in pavucontrol

**Comment by djevertguzman** [comment]
Yep, i have noticed this as well. I tested on my Laptop running Ubuntu, Steam Deck, and Macbook. All 3 on the Steam Beta Client. The Macbook streams audio just fine, but both the Linux clients fail in the same way.

`PulseAudio: Sink alsa_output.pci-0000_04_00.0.hdmi-stereo is default
PulseAudio: Adding sink[937] alsa_output.pci-0000_04_00.0.hdmi-stereo with priority 7, monitor is alsa_output.pci-0000_04_00.0.hdmi-stereo.monitor
PulseAudio: Recording on device alsa_output.pci-0000_04_00.0.hdmi-stereo.monitor
Audio mix: start=430365012633, returned=0
Audio source [System Pulse]: init=0, adjustment=0, through=0, last_start=0, mixed=0, drop_before=0, drop_after=0
PulseAudio: Connected to device alsa_output.pci-0000_04_00.0.hdmi-stereo.monitor (937, not suspended).
PulseAudio: Using sample spec 's16le 2ch 48000Hz', channel map 'front-left,front-right'.
CLIENT: Got control packet k_EStreamControlSetQoS
CLIENT: Got control packet k_EStreamControlSetTargetBitrate
BuildCompleteAppOverviewChange: 756 apps
>>> Switching video stream from NONE to GameOverlay_MovieStream_467056
CLIENT: Starting audio stream: frequency=48000, channels=2
CLIENT: Couldn't open audio device: format=2, channels=2, frequency=48000. Audio is disabled: Source channel map is invalid
`
